### PR TITLE
Changelog for master

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,97 @@
...
+	  into LMDB files (ENT-4484)
+	- 'if' constraint now works in combination with class contexts
+	  (CFE-2615)
+	- Added $(sys.cf_version_release) variable (ENT-5348)
+	- Added between_versions macro to parser (CFE-3198)
+	- Added cf-postgres requirement to cf-apache and cf-hub systemd units
+	  (ENT-5125)
+	- Added else macro in policy parser
+	- Added files promise content attribute (CFE-3276)
+	- Added macros before_version, at_version and after_version (CFE-3198)
```
Comment (craigcomstock) [inline ChangeLog:12]: Would you want to combine the two macros entries which are both CFE-3198?

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,97 @@
...
+	  - cf_version_maximum()
+	  - cf_version_minimum()
+	  - cf_version_after()
+	  - cf_version_before()
+	  - cf_version_at()
+	  - cf_version_between()
+	- Removed (USE AT YOUR OWN RISK) from cf-key help menu for -x (ENT-5090)
+	- Rewrote helloworld.cf to use files promises content attribute
+	  (CFE-3276)
+	- Squash this entry with the one introducing cf-keycrypt (CFE-3272)
```
Comment (craigcomstock) [inline ChangeLog:67]: reword this one? What is the real impact of this entry/commit?

File: ChangeLog [hunk 3]
```diff
@@ -1,3 +1,92 @@
...
+	  (ENT-5125)
+	- Added files promise content attribute (CFE-3276)
+	- Added string_trim() policy function (CFE-3074)
+	- Added warning if CSV parser parses nothing from non-empty file
+	  (CFE-3256)
+	- All changes made by 'files' promises are now reported (ENT-5291)
+	- Bootstrap to loopback interface is now allowed, with a warning
+	  (CFE-3304)
+	- Client initiated reporting was fixed on RHEL 8.1 (ENT-5415)
+	- Directory and file creations are now properly reported as 'info'
```
Comment (nickanderson) [inline ChangeLog:20]: Merge this one into the one above about all files promise changes being reported?

File: ChangeLog [hunk 4]
```diff
@@ -1,3 +1,92 @@
...
+	  messages (ENT-5291)
+	- Failures in edit_xml result in promises marked as failed not
```
Comment (nickanderson) [inline ChangeLog:22]: Also merge into files changes ?

File: ChangeLog [hunk 5]
```diff
@@ -1,3 +1,92 @@
...
+	  - cf_version_at()
+	  - cf_version_between()
+	- Removed (USE AT YOUR OWN RISK) from cf-key help menu for -x (ENT-5090)
+	- Rewrote helloworld.cf to use files promises content attribute
+	  (CFE-3276)
+	- The outcome classes are now defined for the top-level directory when
+	  'include_basedir' is 'false' (ENT-5291)
+	- Variable references with nested parentheses no longer cause errors
+	  (CFE-3242)
+	- Version macros now accept single digits (CFE-3198)
```
Comment (craigcomstock) [inline ChangeLog:68]: maybe combine this one as well? All CFE-3198 on one line?

File: ChangeLog [hunk 6]
```diff
@@ -1,3 +1,92 @@
...
+	- cf-agent: files: report purged dirs and files as repaired (CFE-3260)
```
Comment (nickanderson) [inline ChangeLog:69]: merge into files promise line above?

**Comment by Lex-2008** [comment]
changelog modified as requested!

**Pull request merged**